src/routes/sessions/[id]/+page.svelte

```diff
@@ -202,6 +204,17 @@
 		session.messages = [...session.messages, message];
 	}
 
+	beforeNavigate((navigation) => {
+		if (completion) {
```

Comment:
Add a guard clause:`if (completion) return;`

Reply:
In general I try to add guard clauses in cases where it would reduce the indentation of whatever is inside the `if { .. }` statements.